test/OfflineTest.php

```diff
@@ -109,6 +110,7 @@ public function testConfigObject()
                                  'useStartTls'            => false,
                                  'optReferrals'           => false,
                                  'tryUsernameSplit'       => true,
+                                 'reconnectAttempts'      => 0,
```

Comment:
Does that really need to be set? Because if there's no fallback and that needs to be set that is a Backwards incompatibility! And I'd like to avoid that!